### ISSUE TITLE
Rest create resource release

### ISSUE THREAD

**Issue by yvespp** [liimaorg/liima#172]
I suggest moving the creation of a release form
`POST /resources/{releaseName}`
to 
`POST /resources/{resourceGroupName}
`and the payload only contains the release:
```
{
  "releaseName": "RL-17.10",
  "sourceRelease": "RL-17.04"
}
```
Makes it clearer to use and it is more obvious why two releases have to specified. 
Ref #57

**Comment by RetGal** [comment]
`{
  "releaseName": "RL-17.10",
  "sourceReleaseName": "RL-17.04"
}`

**Comment by RetGal** [comment]
Implemented in pr #177
